src/Accordion/Accordion.tsx

```diff
@@ -1,95 +1,88 @@
 import React from "react";
 import { randomId } from "@sebgroup/frontend-tools/dist/randomId";
-import { isPrimitive } from "@sebgroup/frontend-tools/dist/isPrimitive";
 import classnames from "classnames";
-import { AccordionItemBody } from "./AccordionItemBody";
+import { AccordionItemProps, AccordionItem } from "./AccordionItem";
 import "./accordion.scss";
 
-export interface AccrodionListItem {
-    /** The header of the accordion item */
-    header: React.ReactNode;
-    /** A sub-header description rendered under the header */
-    subHeader?: React.ReactNode;
-    /** The content of the accordion item */
-    content: React.ReactNode;
-}
-
-export interface AccordionProps {
-    /** Element class name */
-    className?: string;
-    /** Element id */
-    id?: string;
+export type AccordionProps = React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement> & {
```

Comment:
Looks like you are including `DetailedHTMLProps` in your accordion props which makes it seem like any of the standard html attributes can be applied to this component which should be mapped accordingly to the html div element but you're not doing that. Or am I missing something? 

For example according to the Typescript definition it is correct for me to do this `<Accordion tabIndex={-1} /> ` but how am I supposed to know that if I pass this value it is never being used?

Reply:
forgot to add `{...props}` to the component. Just added it.